src/main/java/com/bootcamp/demo/security/SecurityConfig.java

```diff
@@ -12,7 +12,7 @@
 
     @Override
     public void configure(HttpSecurity http) throws Exception {
-        http
+        http    .csrf().disable()
```

Comment:
For this exercise you could disable Spring Security altogether if it makes testing/development easier.